### PR TITLE
Changed the app name.

### PR DESCRIPTION
**Pull request by Srishtikumari9**
Added packages.

### CODE REVIEW

File: app/src/main/java/com/example/socialapp/network/Api.java [hunk 1]
```diff
@@ -10,8 +11,12 @@ public interface Api {
...
+                                             @Query("redirect_uri") String redirectUri,
                                              @Query("client_secret") String client_secret);
     @GET("me/posts")
-    Call<Posts> getPosts(@Query("fields") String fields,@Query("access_token") String accessToken);
+    Call<Posts> getPosts(@Query("fields") String fields, @Query("access_token") String accessToken);
+
+   // Address getAddress(int customerId, String customername)
+
 }
+
```
Comment (pawankgupta-se) [inline app/src/main/java/com/example/socialapp/network/Api.java]: We can remove this extra line.

File: app/src/main/java/com/example/socialapp/network/Api.java [hunk 2]
```diff
@@ -10,8 +11,12 @@ public interface Api {
...
     Call<AccessTokenResponse> getAccessToken(@Query("code") String code,
                                              @Query("client_id") String clientId,
-                                       @Query("redirect_uri") String redirectUri,
+                                             @Query("redirect_uri") String redirectUri,
                                              @Query("client_secret") String client_secret);
     @GET("me/posts")
-    Call<Posts> getPosts(@Query("fields") String fields,@Query("access_token") String accessToken);
+    Call<Posts> getPosts(@Query("fields") String fields, @Query("access_token") String accessToken);
+
+   // Address getAddress(int customerId, String customername)
```
Comment (pawankgupta-se) [inline app/src/main/java/com/example/socialapp/network/Api.java]: We can remove commented code if it's not required.

**Review by pawankgupta-se — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/socialapp/network/Api.java (hunk 1 above)
Comment (Srishtikumari9) [reply app/src/main/java/com/example/socialapp/network/Api.java]: Done.

File: app/src/main/java/com/example/socialapp/network/Api.java (hunk 2 above)
Comment (Srishtikumari9) [reply app/src/main/java/com/example/socialapp/network/Api.java]: Done.

**Pull request merged**